src/components/admin/localization-toolkit/LocalizationToolkit.tsx

```
@@ -0,0 +1,75 @@
+import { ReactComponent as GlobeIcon } from '../../../assets/icons/globe.svg';
+import { LocalizationLanguage, TranslationStatusFilter } from '../../../types/common/language';
+import { mapLabelToTranslationStatusFilter } from '../../../utils/functions/mappers/admin/localization-status/localization-status-mappers';
+import './LocalizationToolkit.scss';
+
+export interface LocalizationToolkitProps {
+    languages: LocalizationLanguage[];
+    onLanguageChange: (language: LocalizationLanguage) => void;
+    onTranslationStatusFilterChange: (translationFilter: TranslationStatusFilter | undefined) => void;
+}
+
+export const LocalizationToolkit = ({
+    languages,
+    onLanguageChange,
+    onTranslationStatusFilterChange,
+}: LocalizationToolkitProps) => {
+    const [selectedLanguage, setSelectedLanguage] = useState<LocalizationLanguage | null>(null);
+    const [selectedTranslationStatus, setSelectedTranslationStatus] = useState<TranslationStatusFilter>(
+        TranslationStatusFilter.All,
+    );
+
+    const changeLanguage = useCallback(
+        (language: LocalizationLanguage) => {
+            setSelectedLanguage(language);
+            onLanguageChange(language);
+        },
+        [onLanguageChange],
+    );
+
+    const changeTranslationStatus = useCallback(
+        (translationFilter: TranslationStatusFilter) => {
+            setSelectedTranslationStatus(translationFilter);
+            onTranslationStatusFilterChange(translationFilter);
+        },
+        [onTranslationStatusFilterChange],
+    );
+
+    useEffect(() => {
+        const defaultLanguage = languages.find((language) => language.code === DEFAULT_LOCALE) || languages[0];
+        changeLanguage(defaultLanguage);
+        changeTranslationStatus(TranslationStatusFilter.All);
+    }, [languages, changeLanguage, changeTranslationStatus]);
+
+    return (
+        <div className="toolkit" data-testid="localization-toolkit">
+            {selectedLanguage && (
```

Isn't there a better solution to avoid passing `null` to value? Hiding the ui element if there is no selected language seems off. I understand the intent and that selectedLanguage always has some value and this won't be a problem. But I think it would be better to rely on some default value and have something like "selectedLanguage || defaultLanguage" than hiding the element.

---

The idea behind this was that if no languages were queried - no functionality of seeing different translations should be present. Considering that the select should always have a selected value, that seemed logical at the moment to just hide the select altogether. Didn't want to add additional static data to frontside too